WooCommerce/UITestsFoundation/Screens/Orders/UnifiedOrderScreen.swift

```diff
@@ -136,9 +152,18 @@ public final class UnifiedOrderScreen: ScreenObject {
     /// - Returns: Add Custom Amount screen object.
     public func openAddCustomAmountScreen() throws -> AddCustomAmountScreen {
         addCustomAmountButton.tap()
+        selectFixedAmount()
         return try AddCustomAmountScreen()
     }
 
+    private func selectFixedAmount() {
+        fixedAmountButton.waitAndTap()
+    }
+
+    private func selectPercentageOrderTotal() {
+        percentageOfTotalButton.waitAndTap()
+    }
```

Comment:
since this isn't used in the test, is there a reason to keep this here?

Reply:
Good point. I thought about leaving it handy for the future us, but I (and likely the future me) agree that we shouldn't be adding unused code. =P
Removed in https://github.com/woocommerce/woocommerce-ios/pull/11274/commits/581bcb277977c90d94f0c3b6e748b3799cbf53bc.